src/controllers/helpers.js

```diff
@@ -370,7 +370,15 @@ helpers.trimChildren = function (category) {
 };
 
 helpers.setCategoryTeaser = function (category) {
-	if (Array.isArray(category.posts) && category.posts.length && category.posts[0]) {
+	if (!Array.isArray(category.posts) || !category.posts.length) {
+		return;
+	}
+	let post = category.posts[0];
+	if (post && post.timestamp > Date.now()) {
+		// Filter scheduled topic mainPosts
+		post = category.posts.filter(post => post.timestamp <= Date.now())[0];
```

Comment:
I think this filtering isn't necessary because of the change you make in `src/categories/recentreplies.js`. 

If a topic is scheduled `Categories.updateRecentTidForCid` will not add it to `cid:<cid>:recent_tids` and it won't show up in `category.posts`. Try without the filtering in here and if it works you can remove these.

Reply:
Worked as you've described, removed. :+1: